pkg/gadgets/profile/cpu/tracer/tracer.go

```diff
@@ -254,19 +254,15 @@ func getReport(t *Tracer, kAllSyms []kernelSymbol, stack *ebpf.Map, keyCount key
 	}
 
 	report := types.Report{
-		Node:        t.node,
```

Comment:
From a quick analysis, I saw that removing this assignment makes that the tracer doesn't need to receive the node anymore. I notice it is the same for several gadgets, the ones in the profile and top categories (snapshot is a special case). Did I miss something? Maybe you were planning to do it in another PR?